doc/apps/dgst.pod

```diff
@@ -8,7 +8,7 @@ dgst, sha, sha1, mdc2, ripemd160, sha224, sha256, sha384, sha512, md4, md5 - mes
 
 B<openssl> B<dgst> 
 [B<-help>]
-[B<-sha|-sha1|-mdc2|-ripemd160|-sha224|-sha256|-sha384|-sha512|-md4|-md5>]
+[B<-sha1|-mdc2|-ripemd160|-sha224|-sha256|-sha384|-sha512|-md4|-md5>]
```

Comment:
I meant that maybe since the list will almost always be wrong, we should just use the word I<digest> in the summary?

Not sure.  Don't worry about it :)